plugins/mac-capture/mac-audio.c

```diff
@@ -629,7 +629,8 @@ static bool coreaudio_init(struct coreaudio_data *ca)
 	if (!coreaudio_start(ca))
 		goto fail;
 
-	blog(LOG_INFO, "coreaudio: device '%s' initialized", ca->device_name);
+	blog(LOG_INFO, "coreaudio: Device '%s' [%d Hz] initialized",
+	     ca->device_name, ca->sample_rate);
```

Comment:
Include <inttypes.h>, then change string to:
`"coreaudio: Device '%s' [%" PRIu32 " Hz] initialized"`
instead

Reply:
`PRIu32` resolves to `u` on macOS - given that mac-capture is macOS only, I don't think we necessarily need a portable macro?

Reply:
Fixed

Reply:
Sorry about that, already changed though so either way is fine I suppose. I'll mark as resolved for now